android/src/main/kotlin/io/github/itzmeanjan/intent/IntentPlugin.kt

```diff
@@ -88,6 +88,8 @@ class IntentPlugin(private val registrar: Registrar, private val activity: Activ
             "startActivity" -> {
                 val intent = Intent()
                 intent.action = call.argument<String>("action")
+                if (call.argument<String>("package") != null)
```

Comment:
This it ok, but required implementation on dart side is not added. 

Otherwise one won't be able to set `package` param from dart code.

I gonna do that.

Thanks for this @togiberlin 

:blush: